plugins/oaiMetadataFormats/dc/locale/ar_IQ/locale.po

```diff
@@ -1,18 +0,0 @@
-msgid ""
```

Comment:
These locale files already exist in the `lib/pkp/plugins/oaiMetadataFormats/dc/locale` folder.